### PR TITLE
Add a traefik example

### PR DESCRIPTION
**Pull request by pavilo**
It took me a while to get Woodpecker running with Traefik. I want to contribute a working example configuration.

### CODE REVIEW

File: docs/docs/30-administration/70-proxy.md [hunk 1]
```diff
@@ -105,3 +105,75 @@ ngrok http 8000
...
+## Traefik
+
+To install the Woodpecker server behind a [traefik](https://traefik.io/) load balancer, you must expose both the `http` and the `gRPC` ports. Here is a comprehensive example, considering you are running traefik with docker swarm and want to do TLS termination and automatic redirection from http to https.
+
+```yml
+version: '3.8'
+
+services:
+  server:
+    image: woodpeckerci/woodpecker-server:v0.15.3
```
Comment (6543) [inline docs/docs/30-administration/70-proxy.md:119]:
```suggestion
    image: woodpeckerci/woodpecker-server:latest
```

Comment (6543) [reply docs/docs/30-administration/70-proxy.md:119]: else we have to update docs on every release

File: docs/docs/30-administration/70-proxy.md [hunk 2]
```diff
@@ -105,3 +105,75 @@ ngrok http 8000
...
+        - traefik.http.routers.woodpecker-secure.entrypoints=websecure
+        - traefik.http.routers.woodpecker-secure.service=woodpecker-service
+
+        - traefik.http.routers.woodpecker.rule=Host(`cd.yourdomain.com`)
+        - traefik.http.routers.woodpecker.entrypoints=web
+        - traefik.http.routers.woodpecker.service=woodpecker-service
+
+        - "traefik.http.middlewares.woodpecker-redirect.redirectscheme.scheme=https"
+        - "traefik.http.middlewares.woodpecker-redirect.redirectscheme.permanent=true"
+        - "traefik.http.routers.woodpecker.middlewares=woodpecker-redirect@docker"
```
Comment (6543) [inline docs/docs/30-administration/70-proxy.md:149]:
```suggestion
        - traefik.http.middlewares.woodpecker-redirect.redirectscheme.scheme=https
        - traefik.http.middlewares.woodpecker-redirect.redirectscheme.permanent=true
        - traefik.http.routers.woodpecker.middlewares=woodpecker-redirect@docker
```
?

File: docs/docs/30-administration/70-proxy.md [hunk 3]
```diff
@@ -105,3 +105,75 @@ ngrok http 8000
...
+        - traefik.http.routers.woodpecker-grpc-secure.entrypoints=websecure
+        - traefik.http.routers.woodpecker-grpc-secure.service=woodpecker-grpc
+
+        - traefik.http.routers.woodpecker-grpc.rule=Host(`woodpecker-grpc.yourdomain.com`)
+        - traefik.http.routers.woodpecker-grpc.entrypoints=web
+        - traefik.http.routers.woodpecker-grpc.service=woodpecker-grpc
+
+        - "traefik.http.middlewares.woodpecker-grpc-redirect.redirectscheme.scheme=https"
+        - "traefik.http.middlewares.woodpecker-grpc-redirect.redirectscheme.permanent=true"
+        - "traefik.http.routers.woodpecker-grpc.middlewares=woodpecker-grpc-redirect@docker"
```
Comment (6543) [inline docs/docs/30-administration/70-proxy.md:167]:
```suggestion
        - traefik.http.middlewares.woodpecker-grpc-redirect.redirectscheme.scheme=https
        - traefik.http.middlewares.woodpecker-grpc-redirect.redirectscheme.permanent=true
        - traefik.http.routers.woodpecker-grpc.middlewares=woodpecker-grpc-redirect@docker
```
?

**Review by 6543 — APPROVED** [review]

**Comment by dscottboggs** [comment]
>  you must expose both the `http` and the `gRPC` ports.

why? None of the other proxies require it

> You should pass `WOODPECKER_GRPC_SECURE=true` and `WOODPECKER_GRPC_VERIFY=true` to your agent when using this configuration.

this seems like it should be more prominent if gRPC *really* needs to be exposed.

**Comment by 6543** [comment]
Well please dont comment in old closed/merged issues/pulls

I think this is worth a new issue or a pull.

PS: i would just leave the example as advanced and add the simple upfront

**Pull request merged**